fr-FR/lang/numere.nlng

```diff
@@ -0,0 +1,1186 @@
+PARSERFUNCS_LISTFUNC_FUNC_TANH_[NUM_HYPERBOLIC]=			tanh(x)                                                     VAL      - Tangente hyperbolique de x. %%DEFINED_FOR_ALL%%
+PARSERFUNCS_LISTFUNC_FUNC_TAN_[NUM_TRIGONOMETRIC]=			tan(x)                                                      VAL      - Tangente de x. %%DEFINED_FOR_ALL_C_RANGE_R%% R \ {-n*PI/2, n*PI/2}. n est un entier naturel > 0.
+PARSERFUNCS_LISTFUNC_FUNC_TEXTPARSE_[STRING]=               textparse(sToParse,sPattern,nPos1=1,nPos2=sToParse.len)    {ARG}     - Décompose sToParse en différents blocs de différents types de variables à l'aide du schéma sPattern. Les types de variables sont définis par les identificateurs suivants dans la chaîne de caractères sPattern:%%ITEMIZE%%%s : chaîne de caractères%%ITEMIZE%%%f : nombre réel ou complexe%ITEMIZE%%%t : valeur de date et d'heure en UTC (dans les formats acceptés par les tables)%%ITEMIZE%%%l : valeur logique%%ITEMIZE%%%L : Valeur numérique formatée en LaTeX%%ITEMIZE%%h : valeur hexadécimale%%ITEMIZE%%o : valeur octale%%ITEMIZE%%b : valeur binaire%%ITEMIZE%%a : valeur arbitraire, est ignorée et sert à sauter une partie du texte%%ITEMIZE_END%%Exemple pour sPattern : "TIME=%t MSG=%s VAL=%f". Les paramètres nPos1 et nPos2 sont facultatifs et indiquent les points de départ et de fin
+PARSERFUNCS_LISTFUNC_FUNC_TEXT_[DRAW]=						text(P,P,TEXT,STYLE)                                        {}       - Imprime le texte TEXT à l'emplacement P. L'alignement et la couleur peuvent être influencés par STYLE, l'alignement et la forme devant être séparés de la couleur par :. Les couleurs du texte sont déterminées comme toutes les autres couleurs selon "help colors" ; l'alignement par LRC (Left-Right-Centered) et la forme par rib (roman-italic-bold). La deuxième paire de coordonnées optionnelle peut être utilisée si le texte ne doit pas être imprimé horizontalement.
+PARSERFUNCS_LISTFUNC_FUNC_THETA_[ANGULAR_COORDS]=			theta(x,y,z)                                                VAL      - Calcule l'angle entre l'axe z et le vecteur (x,y,z). %%DEFINED_FOR_ALL%%
+PARSERFUNCS_LISTFUNC_FUNC_TIMEFORMAT_[TIME_STRING]=			timeformat(sFormat,fTimeVal)                                STR      - Formate l'heure fTimeVal selon le schéma sFormat. Si le schéma est une chaîne de caractères vide, le format interne par défaut est utilisé. Les caractères suivants dans sFormat sont remplacés par la valeur temporelle correspondante en fonction de leur longueur :%%ITEMIZE%%Y : année%%ITEMIZE%%M : mois%%ITEMIZE%D : jour (comme valeur de date)%%ITEMIZE%d : jour (comme durée)%%ITEMIZE%H : Heure (en UTC)%%ITEMIZE%h : heure (dans le fuseau horaire actuel)%%ITEMIZE%m : minute%%ITEMIZE%%s : seconde%%ITEMIZE%i : milliseconde%%ITEMIZE%u : microseconde%ITEMIZE_END%%%Exemple : "YY-MM-DD HH:mm:ss"
```

Comment:
The last `%%ITEMIZE_END%%` was replaced with `%ITEMIZE_END%% %`e erroneously. A possible approach could be to replace the `%%SOMETHING%%` tokens with e.g. `%_1_%` (with ascending numbers per string), store them, translate the string and then replace it back. Do *not*  use `%%1%%`, because that has another meaning.